src/Navbar.jsx

```diff
@@ -34,6 +35,7 @@ function Navbar({showCountdown}) {
             {showCountdown && <CountdownTimer />}
             {showStats && <ModalStats setIsOpen={setShowStats} />}
             {showInstructions && <ModalInstructions setIsOpen={setShowInstructions} />}
+            <input id={styles.songSelector} type="number" name="choose song" placeholder="Enter Song Number" min="1" step="1" max={maxSongSelect.toString()} onChange={handleAnswer}></input>
```

Comment:
Another thought: we should default to the current song ID, so that people can toggle up or down and don't have to guess a number if they want to play the previous day's song

Reply:
We can replace the placeholder with the ID of today's song

Reply:
Sure, not against the idea. I'm not sure what styling you want though, so do you want to handle the material ui bit?

If we do use the ID of today's song as the placeholder we should also give an indicator, maybe a label, of what the number is for.